client/src/app/app-routing.module.ts

```diff
@@ -45,7 +45,8 @@ const routes: Routes = [
 ];
 
 @NgModule({
-	imports: [RouterModule.forRoot(routes)],
+	imports: [AppRoutingModule,
```

Comment:
Why was `AppRoutingModule` imported here?

Reply:
It was a mistake. AppRoutingModule should be import to **module.ts** not to the **app-routing-module.ts** I will update it.